### ISSUE TITLE
One diode / valve reversed?

### ISSUE THREAD

**Issue by tom-f-oconnell** [FlySorterLLC/MAPLEHardware#8]
It seems that if you were to follow the instructions: "the white stripe of the diodes should be on the left with the board in the orientation shown"

...D2, the diode for the small part manipulator air FET, would be connected incorrectly, considering the polarity of that output is reversed, at least on my version of the Smoothieboard. Maybe the schematics are right and this is just an instruction issue, or maybe I'm just misunderstanding. If the different direction of the polarity is just a Smoothieboard version thing, then you might need to just specify to switch that diode depending on the version.

If this is true, and if the valves do have polarity, than I suppose that should be switched downstream. It seems this model of valve isn't polarized (i.e. internal diode protection, from what I understand), but it that was just from taking some measurements, as the datasheets did not seem to make it clear whether this part number was polarized. The wires are colored, so it at least had me worried.

Not directly related, but it seems some combination of this and a possible short might have broken some of my FETs in early testing. With my soldering, which maybe could have been cleaner, the 2mm spacers under the smaller circuit board were insufficient to prevent some shorts through the aluminum Z plate. It might be worth mentioning things should stay well under 2mm in the soldering part of the instructions, or recommending thicker spacers.

**Comment by flysorter** [comment]
Good catch! You're right that my PCB doesn't account for the fact that the polarities of the switch circuitry is reversed between D1 and D2. Not the first time I've been frustrated that the Smoothie's outputs are not always intuitive (even if they are labeled on the board).

I will update the PCBs at some point, and the instructions as well. Thanks much.

**Comment by flysorter** [comment]
As I think about this more, it may be that this error did in fact lead to your FET failure. Sorry for that!

The board rework isn't super complicated, but I will write up some instructions for that, too (since you won't want to wait for a new set of PCBs, nor will you have to).

**Comment by tom-f-oconnell** [comment]
If the rework instructions would just be for my benefit, I've already figured out what to do there, but thanks.
